eng/scripts/Initialize-VcpkgRelease.ps1

```diff
@@ -37,7 +37,7 @@ param (
 # If there's nothing in the "port" folder to upload set SkipVcpkgUpdate to true
 # and exit. Other steps will check SkipVcpkgUpdate to decide whether to move
 # forward.
-if (!(Get-ChildItem -Path "$SourceDirectory/port/CONTROL")) {
+if (!(Get-ChildItem -Path "$SourceDirectory/port/vcpkg.json")) {
```

Comment:
@danieljurek, can you fork off this branch https://github.com/ahsonkhan/azure-sdk-for-cpp/tree/UpdatePorts to make the engsys fixes in the release pipeline, if we need to make any other fixes, and test it with the daily vcpkg test PR.

Let me know if you want me to keep this change in as part of this PR, or remove it.